### PR TITLE
ADD -disableNgsiv1 and -logDeprecate CLIs

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

File: doc/manuals/admin/logs.md [hunk 1]
```diff
@@ -216,6 +216,15 @@ time=2020-10-22T19:51:03.601Z | lvl=INFO | corr=eabce3e2-149f-11eb-a2e8-000c29df
 time=2020-10-22T19:51:03.602Z | lvl=INFO | corr=eabce3e2-149f-11eb-a2e8-000c29df7908 | trans=1603396258-520-00000000006 | from=0.0.0.0 | srv=s1 | subsrv=/A | comp=Orion | op=logTracing.cpp[79]:logInfoRequestWithoutPayload | msg=Request received: GET /v2/entities/E1?type=T1, response code: 200
 ```
 
+* If `-logDeprecate` is used, then NGSIv1 requests (both with and without payload) are logged at WARN level (note this doesn't
+  include notifications using [`"attrsFormat": "legacy"`](../orion-api.md#subscriptionnotification) or forward requests corresonding
```
Comment (fgalan) [inline doc/manuals/admin/logs.md:220]:
```suggestion
  include notifications using [`"attrsFormat": "legacy"`](../orion-api.md#subscriptionnotification) or forward requests corresponding
```

File: doc/manuals/admin/cli.md [hunk 2]
```diff
@@ -178,9 +178,13 @@ The list of available options is the following:
 -   **-logInfoPayloadMaxSize**. For those log traces at INFO level that print request and/or response payloads, this is the maximum allowed size for those payloads. If the payload size is greater than this setting, then only the first `-logInfoPayloadMaxSize` bytes are included (and an ellipsis in the form of `(...)` is shown in trace). Default value: 5 KBytes. It can be changed after Orion startup with the [log admin REST API](management_api.md#log-configs-and-trace-levels), with the `infoPayloadMaxSize` field.
 -   **-disableMetrics**. To turn off the 'metrics' feature. Gathering of metrics is a bit costly, as system calls and semaphores are involved.
     Use this parameter to start the broker without metrics overhead.
+-   **-disableNgsiv1**. To turn off NGSIv1 operations. Note that only API endpoints are disabled, notifications using
+    [`"attrsFormat": "legacy"`](../orion-api.md#subscriptionnotification) or forward requests corresonding to registrations
```
Comment (fgalan) [inline doc/manuals/admin/cli.md:182]:
```suggestion
    [`"attrsFormat": "legacy"`](../orion-api.md#subscriptionnotification) or forward requests corresponding to registrations
```

**Review by mapedraza — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -2,6 +2,8 @@
 - Add: subject.condition.notifyOnMetadataChange field to subscriptions, so only-metadata changes doesn't trigger notifications (#3727)
 - Add: new simplifiedNormalized and simplifiedKeyvalues for attrsFormat (#4286)
 - Add: -mqttTimeout (env var ORION_MQTT_TIMEOUT) for MQTT broker connection timeout
+- Add: -disableNgsiv1 (env var ORION_DISABLE_NGSIV1) to disable NGSIv1 API requests
+- Add: -logDeprecate (env var ORION_LOG_DEPRECATE) to log deprecate usages at WARN level
```
Comment (mapedraza) [inline CHANGES_NEXT_RELEASE:6]: Should other deprecated stuff be sensible too to this CLI?
https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/deprecated.md

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:6]: Yes, the idea is that `-logDeprecated` gets extended with new deprecation usages beyond NGSIv1 requests (but not in this PR :)

**Comment by fisuda** [comment]
I sent the PR #4342 to this branch. Thanks.

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**